stdlib/_operator.pyi

```diff
@@ -25,6 +24,24 @@ _K = TypeVar("_K")
 _V = TypeVar("_V")
 _P = ParamSpec("_P")
 
+# The following protocols return "Any" instead of bool, since the comparison
+# operators can be overloaded to return an arbitrary object. For example,
+# the numpy.array comparison dunders return another numpy.array.
+
+class _SupportsDunderLT(Protocol):
+    def __lt__(self, __other: Any) -> Any: ...
+
+class _SupportsDunderGT(Protocol):
+    def __gt__(self, __other: Any) -> Any: ...
+
+class _SupportsDunderLE(Protocol):
+    def __le__(self, __other: Any) -> Any: ...
+
+class _SupportsDunderGE(Protocol):
+    def __ge__(self, __other: Any) -> Any: ...
+
+_SupportsAnyComparison = _SupportsDunderLE | _SupportsDunderGE | _SupportsDunderGT | _SupportsDunderLT
```

Comment:
I've actually sort of come to regret the name I gave this Union in `_typeshed`. `SupportsAnyComparison` implies that "any comparison method will work" for an object satisfying this interface. But that's not what this interface actually represents.

`SupportsSomeComparison` might be a better name.

```suggestion
_SupportsSomeComparison = _SupportsDunderLE | _SupportsDunderGE | _SupportsDunderGT | _SupportsDunderLT
```

Reply:
Or just `_SupportsComparison`